src/scripts.js

```diff
@@ -1 +1,30 @@
-console.log("Hello World");
\ No newline at end of file
+// ~~~~~ QUERY SELECTORS ~~~~~
```

Comment:
Our only question here, I think is do we need to `import` different files to make our linter work?